canonical-web-and-design/snapcraft.io
# Annotate all the yamls

![](https://i.imgflip.com/30jnzo.jpg)

Fixes #1897
Adds interactive YAML viewer to all tutorials of first snap flow.

### QA

- ./run or https://snapcraft-io-canonical-web-and-design-pr-1900.run.demo.haus/
- go to first snap flow
- all existing language tutorials should have interactive yaml viewer and ability to download generated snapcraft.yaml
